### ISSUE TITLE
Pick gives cryptic terminal warnings at startup

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#707]
Workflow:

* Open Ginga 2.7.2.
* Load `Pick` plugin (with or without opening up any image).

I see this on the terminal but nothing on the log (attached):

```
** (python:25221): WARNING **: 16:38:20.602: invalid source position for vertical gradient

** (python:25221): WARNING **: 16:38:20.602: invalid source position for vertical gradient

** (python:25221): WARNING **: 16:38:20.606: invalid source position for vertical gradient

** (python:25221): WARNING **: 16:38:20.606: invalid source position for vertical gradient
```

[ginga.log](https://github.com/ejeschke/ginga/files/2563771/ginga.log)

**Comment by ejeschke** [comment]
Meaning this error pops up when you start `Pick`?  Wonder whether that is coming from Qt or Matplotlib or ?

**Comment by pllim** [comment]
Yes, it pops out when I start Pick. I don't know what is producing it. How do I find out?

**Comment by ejeschke** [comment]
@pllim, I don't think this is related to a bug in `Ginga` code particularly.  Try searching "python invalid source position for vertical gradient" and you'll see what I mean.

**Comment by ejeschke** [comment]
If you resize the `Pick` window, or the whole reference viewer, does the error go away?
